docs/Reference/Limits.md

```diff
@@ -1,51 +1,74 @@
+We have a limit of 35
+[included files](../Project-Configuration/Project-Configuration-Files/#Include)
+per project based on this
+[investigation](https://jira.mongodb.org/browse/DEVPROD-3509) about GitHub
+usage.
 
 ## YAML configuration size
 
-Large parser projects are disabled, which means that Evergreen limits YAML project configuration sizes to 16MB.
-The 16 MB limit is on the sum of the size of the generate.tasks input file and the size of the project configuration YAML. If the YAML length > 16 MB after task generators have tacked on their configuration to the project configuration YAML, the task will fail.
+Large parser projects are disabled, which means that Evergreen limits YAML
+project configuration sizes to 16MB. The 16 MB limit is on the sum of the size
+of the generate.tasks input file and the size of the project configuration YAML.
+If the YAML length > 16 MB after task generators have tacked on their
+configuration to the project configuration YAML, the task will fail.
 
 #### Can an exception be requested?
 
-This cannot be lifted because Evergreen cannot safely handle larger file sizes from multiple users. Users are encouraged to instead try creating a new patch with fewer variants/tasks.
+This cannot be lifted because Evergreen cannot safely handle larger file sizes
+from multiple users. Users are encouraged to instead try creating a new patch
+with fewer variants/tasks.
 
 ## Patch size
 
-Evergreen has a 100MB system limit on patch size (the diff size for the changes). In order to submit patches that are larger than 16MB from the CLI, the `--large` flag needs to be used. Large patches may hit the CLI’s one minute timeout. To work around that, users can open a PR and run the patch from there.
+Evergreen has a 100MB system limit on patch size (the diff size for the
+changes). In order to submit patches that are larger than 16MB from the CLI, the
+`--large` flag needs to be used. Large patches may hit the CLI’s one minute
+timeout. To work around that, users can open a PR and run the patch from there.
 
 ## What is the largest file size I can upload to Parsley?
 
-Evergreen does not have a limit for Parsley file sizes. However, you will encounter a limit based on the browser that you are using. As of April 4, 2022 the limits are:
-
-In V8 (used by Chrome and Node), the maximum length is 229 - 24 (~1GiB). On 32-bit systems, the maximum length is 228 - 16 (~512MiB). In Firefox, the maximum length is 230 - 2 (~2GiB). Before Firefox 65, the maximum length was 228 - 1 (~512MiB). In Safari, the maximum length is 231 - 1 (~4GiB).
+Parsley will not accept files larger than 2.5GB. If you need to upload a file
+larger than 2.5GB, we recommend downloading the raw file and splitting the file
+into smaller parts and uploading them separately.
```

Comment:
just a suggestion because the sentence has double `and`s which could read awkwardly
```suggestion
into smaller parts, then uploading them separately.
```